code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -85,6 +114,39 @@
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
 	update_mob()
 	explosion(get_turf(src.loc),-1,0,2)
+	if(shrapnel_list.len > 0)
+		var/i
+		var/atom/target
+		var/list/bodyparts = list("head","chest","groin","l_arm","r_arm","l_hand","r_hand","l_leg","r_leg","l_foot","r_foot")
+		for(i=1, i<=shrapnel_list.len, i++)
+			var/obj/item/shrapnel = shrapnel_list[i]
+			if(shrapnel.is_shrapnel)
+
+				var/amount = shrapnel.shrapnel_amount
+
+				while(amount > 0)
+					amount = amount - 1
```

Comment:
`amount--` or `--amount`
If you want to be EXTRA fancy, you can even fold this into the previous line by changing it to `while(amount-- > 0)`

(If you do that, it has to be `amount--` rather than `--amount`. The former returns the value THEN decrements, while the latter decrements first. **The More You Know**:rainbow::star:)